src/tile.scss

```diff
@@ -1,97 +1,657 @@
+        @include set-height(2rem);
+        @include set-width(2rem);
+        @include fd-flex-center();
+
+        background-color: var(--sapBlockLayer_Background);
+        opacity: 0.6;
+        border-radius: 50%;
+        font-family: SAP-icons;
+        font-size: 1rem;
+        color: var(--sapButton_Emphasized_TextColor);
+        text-shadow: var(--sapButton_Emphasized_TextShadow);
+        content: "\e14b";
+      }
+
+      &--pause {
+        &::before {
+          content: "\e14c";
+        }
+      }
+
+      @include fd-focus() {
+        outline: none;
+      }
+
+      @include fd-hover() {
+        &::before {
+          opacity: 0.8;
+        }
+      }
+
+      @include fd-active() {
+        &::before {
+          opacity: 1;
+        }
+      }
+    }
+
+    .#{$block}__container {
+      @include fd-reset();
+      @include set-height(6.125rem);
+
+      @include fd-flex($direction: column) {
+        justify-content: space-between;
+      }
+
+      z-index: 2;
+      width: 100%;
+      max-width: 100%;
+      background-color: var(--sapBlockLayer_Background);
+      opacity: 0.8;
+      padding-top: 0.375rem;
+      padding-left: $fd-tile-padding;
+      padding-right: $fd-tile-padding;
+      border-bottom-left-radius: $fd-tile-border-radius;
+      border-bottom-right-radius: $fd-tile-border-radius;
+    }
+
+    .#{$block}__header {
+      @include set-height(3.5rem);
+    }
+
+    .#{$block}__title,
+    .#{$block}__subtitle,
+    .#{$block}__footer-text {
+      color: var(--sapContent_ContrastTextColor); // TO BE CONFIRMED
+      text-shadow: var(--sapContent_ContrastShadowColor); // TO BE CONFIRMED
+      font-size: var(--sapFontSize);
+      line-height: normal;
+    }
+
+    .#{$block}__page-indicator {
+      @include fd-reset();
+      @include fd-flex-center();
+      @include set-height(0.5rem);
+
+      margin-bottom: 0.125rem;
+      margin-top: 0.375rem;
+    }
+
+    .#{$block}__dot {
+      @include fd-reset();
+      @include set-height($fd-tile-indicator-dot-size);
+      @include set-width($fd-tile-indicator-dot-size);
+
+      border-radius: 50%;
+      background-color: var(--sapContent_ContrastTextColor); // TO BE CONFIRMED
+      box-shadow: 0 0 0.25rem var(--sapContent_ShadowColor);
+      margin-right: $fd-tile-indicator-dot-spacing;
+
+      @include fd-rtl() {
+        margin-right: 0;
+        margin-left: $fd-tile-indicator-dot-spacing;
+      }
+
+      &--active {
+        @include set-height($fd-tile-indicator-dot-active-size);
+        @include set-width($fd-tile-indicator-dot-active-size);
+
+        color: var(--sapContent_ContrastTextColor); // TO BE CONFIRMED
+      }
+    }
+
+    &.#{$block}--s {
+      .#{$block}__container {
+        @include set-height(5.25rem);
+
+        padding-top: 0.25rem;
+        padding-right: $fd-tile-small-padding;
+        padding-left: $fd-tile-small-padding;
+      }
+
+      .#{$block}__header {
+        @include set-height(3.188rem);
```

Comment:
It shouldn't be 3.25rem ?
![image](https://user-images.githubusercontent.com/26483208/82341369-a359e600-99f0-11ea-8dad-4eed8022c494.png)
![image](https://user-images.githubusercontent.com/26483208/82341390-a94fc700-99f0-11ea-8376-6194e20fe0d3.png)